mobile/src/bundles/auth/components/auth-wrapper/styles.ts

```diff
@@ -1,5 +1,4 @@
-import { StyleSheet } from 'react-native';
-
+import { StyleSheet } from '~/bundles/common/components/components';
```

Comment:
I think it's better to get if from styles folder

Reply:
You mean left import { StyleSheet } from 'react-native'; in styles.ts?